ci_scripts/ci_post_clone.sh

```diff
@@ -68,9 +31,48 @@ APP_ID_IOS = org.mozilla.ios.FirefoxVPN
 NETEXT_ID_IOS = org.mozilla.ios.FirefoxVPN.network-extension
 EOF
 
-if [ $CI_PRODUCT_PLATFORM == 'macOS' ]
-then
-  ./scripts/macos/apple_compile.sh macos
-else
-  ./scripts/macos/apple_compile.sh ios -q /Volumes/workspace/repository/qt_ios/$QTVERSION/macos/bin
-fi
+# TODO: REVERT HARDCODED LINK HERE
```

Comment:
I'm going to send a follow up for this. We need to merge as is, and then on ce the toolchain job runs on CI we can change.